app/views/finance/balancing/new.html.haml

```diff
@@ -53,13 +53,16 @@
       class: 'btn'
   .btn-group
     - unless @order.closed?
-      = link_to t('.confirm_order'), confirm_finance_order_path(@order), class: 'btn btn-primary'
-      = link_to '#', data: {toggle: 'dropdown'}, class: 'btn dropdown-toggle' do
-        %span.caret
-      %ul.dropdown-menu
-        %li= link_to t('.confirm_order'), confirm_finance_order_path(@order)
-        %li= link_to t('.close_direct'), close_direct_finance_order_path(@order), method: :patch,
-          data: {confirm: t('.close_direct_confirm')}
+      - if FoodsoftConfig[:charge_members_manually]
+        = link_to t('.confirm_order'), close_direct_finance_order_path(@order), class: 'btn btn-primary', method: :patch
+      - else
+        = link_to t('.confirm_order'), confirm_finance_order_path(@order), class: 'btn btn-primary'
+        = link_to '#', data: {toggle: 'dropdown'}, class: 'btn dropdown-toggle' do
+          %span.caret
+        %ul.dropdown-menu
+          %li= link_to t('.confirm_order'), confirm_finance_order_path(@order)
+          %li= link_to t('.close_direct'), close_direct_finance_order_path(@order), method: :patch,
+            data: {confirm: t('.close_direct_confirm')}
```

Comment:
Maybe we better remove the close direct option for automatic charging in a future version.

Reply:
IMHO it's still usefull sometmes, when e.g. a ordner wasn't deliverd and nobody should get charged.